packages/xrpl/src/models/methods/accountInfo.ts

```diff
@@ -180,3 +171,58 @@ export interface AccountInfoResponse extends BaseResponse {
     validated?: boolean
   }
 }
+
+/**
+ * Response expected from a {@link AccountInfoRequest}.
+ *
+ * @category Responses
+ */
+export interface AccountInfoResponse extends BaseAccountInfoResponse {
+  result: BaseAccountInfoResponse['result'] & {
+    /**
+     * If requested, array of SignerList ledger objects associated with this account for Multi-Signing.
+     * Since an account can own at most one SignerList, this array must have exactly one
+     * member if it is present.
+     */
+    signer_lists?: SignerList[]
+  }
+}
+
+/**
+ * Response expected from a {@link AccountInfoRequest} using API version 1.
+ *
+ * @category Responses
```

Comment:
Maybe we should also add an @category for `v1` or otherwise make it so our output docs render better to tag things as `v1`

I would utilize both `@since` for api v2 functions and `@version` for API v1 stuff.

see: https://jsdoc.app/

Reply:
Sure, I can change it to `@category ResponsesV1` for v1 types and leave `@category Responses` for v2.

Reply:
However, `@since` and `@version` require version values such as `2.0.0` so `v2` wouldn't think. I think this could confuse users thinking `2.0.0` means client library version and not rippled.